packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -3,78 +3,100 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import { ComponentPropsWithoutRef } from "react";
+import { ComponentPropsWithoutRef, forwardRef, ForwardRefExoticComponent, RefAttributes } from "react";
 import { Anchor, Text, Tree } from "@itwin/itwinui-react/bricks";
 import { MAX_LIMIT_OVERRIDE } from "../internal/Utils.js";
 import { PresentationInfoNode } from "../TreeNode.js";
-import { useTree } from "../UseTree.js";
+import { HierarchyLevelDetails, useTree } from "../UseTree.js";
 import { FlatNode } from "./FlatTreeNode.js";
 import { useLocalizationContext } from "./LocalizationContext.js";
-import { TreeNodeRenderer } from "./TreeNodeRenderer.js";
+
+/** @alpha */
+export interface TreeErrorActionsProps {
```

Comment:
May this could be renamed to `TreeErrorItemProps`

Reply:
renamed